modules/shortcodes/wordads.php

```diff
@@ -53,14 +57,49 @@ static function wordads_shortcode_html( $atts, $content = '' ) {
 			return '<div>' . __( 'The WordAds module is not active', 'jetpack' ) . '</div>';
 		}
 
-		$html = '<div class="jetpack-wordad" itemscope itemtype="https://schema.org/WPAdBlock">';
-
-		$html .= '</div>';
-
+		$html = '<div class="jetpack-wordad" itemscope itemtype="https://schema.org/WPAdBlock"></div>';
 		$html = $wordads->insert_inline_ad( $html );
 
 		return $html;
 	}
+
+	public static function gutenblock_render( $attr ) {
+		global $wordads;
+
+		/**
+		 * Allow third-party tools to disable the display of in post ads.
+		 *
+		 * @module wordads
+		 *
+		 * @since 4.5.0
+		 *
+		 * @param bool true Should the in post unit be disabled. Default to false.
+		 */
+		if ( is_feed() || apply_filters( 'wordads_inpost_disable', false ) ) {
+			return '';
+		}
+
+		if ( $wordads->option( 'wordads_house' ) ) {
+			return $wordads->get_ad( 'inline', 'house' );
+		}
+
+		$section_id = 0 === $wordads->params->blog_id ? WORDADS_API_TEST_ID : $wordads->params->blog_id . '6'; // 6 is to keep track of gutenblock ads
+		$align = 'center';
+		if ( isset( $attr['align'] ) && in_array( $attr['align'], array( 'left', 'center', 'right' ) ) ) {
+			$align = $attr['align'];
+		}
+		$align = 'align' . $align;
+
+		$format = 'mrec';
+		if ( isset( $attr['format'] ) && in_array( $attr['format'], array_keys( WordAds::$ad_tag_ids ) ) ) {
+			$format = $attr['format'];
+		}
+
+		$height = WordAds::$ad_tag_ids[$format]['height'];
+		$width = WordAds::$ad_tag_ids[$format]['width'];
+		$snippet = $wordads->get_ad_snippet( $section_id, $height, $width, 'inline', 'inpost', 'float:left;margin-right:5px;margin-top:0px;' );
```

Comment:
Do we always expect styles to be `float:left;margin-right:5px;margin-top:0px;`? 
What about when alignment is something else what about RTL?

Reply:
This is just something to provide a little room for the "report ad" link which gets added dynamically from the ad JS.